### ISSUE TITLE
Microsoft acquiring GitHub

### ISSUE THREAD

**Issue by IMRC21** [Microsoft/vscode#51091]
Steps to Reproduce:

1. Find 7.5B $
2. Buy GitHub

**Comment by Gatgeagent** [comment]
You are totally right.
This certainly has to get fixed asap!

**Comment by ahmedsubkhi** [comment]
Ganas...

**Comment by dekzitfz** [comment]
can we close this issue now? :D

**Comment by Kompwu** [comment]
America, Fuck Yeah !

**Comment by YoungClusterM** [comment]
@dekzitfz No, we can't close it now - It's the hugest issue that had ever been

**Comment by gamexxx** [comment]
Ohhhh, nooo.. Please, stop, Microsoft!

**Comment by SanSYS** [comment]
https://github.com/martenbjork/github-xp - extension for chrome makes your github like XP

![image](https://user-images.githubusercontent.com/5781038/40996044-4e876e76-691a-11e8-99b2-b1514dd2b18a.png)
